### PR TITLE
OpenAPI 3 fixes: definitions => components/schemas + parameters need …

### PR DESCRIPTION
**Pull request by burgalon**
…`schema`

### CODE REVIEW

File: docs/docs/adding-swagger.md [hunk 1]
```diff
@@ -16,9 +16,15 @@ To add swagger to a `remult` application follow these steps:
    
    const app = express();
    let api = remultExpress();
+   const swaggerDocument = api.openApiDoc({ title: 'remult-react-todo' });
+   const options = {
+    swaggerOptions: {
```
Comment (noam-honig) [inline docs/docs/adding-swagger.md:21]: What is the advantage of creating this options object, instead of just sending the document to swaggerUi.setup?

**Comment by burgalon** [comment]
in order to serve the swagger JSON itself

**Comment by burgalon** [comment]
see here - https://www.npmjs.com/package/swagger-ui-express

File: docs/docs/adding-swagger.md (hunk 1 above)
Comment (burgalon) [reply docs/docs/adding-swagger.md:21]: in order to serve the swagger JSON itself
 see here https://www.npmjs.com/package/swagger-ui-express

**Comment by noam-honig** [comment]
Thanks

**Pull request merged**